javascript-forms/form-values.js

```diff
@@ -0,0 +1,23 @@
+var contactForm = document.querySelector('#contact-form');
+// console.log(contactForm);
+
+var valuesObject = {};
```

Comment:
You should move this inside the function, it doesn't need to continue to exist after the function ends.